src/Service/Core.php

```diff
@@ -234,6 +237,10 @@ public function getAppNode($module, $language = NULL) {
       $nid = end($nids);
```

Comment:
Any reason why the $query can't be modified to load the last $nid instead?

Reply:
good question, probably not (apart from not having the time to change it...)